pkgs/jupiter-hw-support/automount-fix-system-paths.patch

```diff
@@ -7,7 +7,7 @@ index 386f4d8..cf79351 100755
        # TODO use -ifrunning and check return value - if there was a steam process and it returns -1, the message wasn't sent
        # need to retry until either steam process is gone or -ifrunning returns 0, or timeout i guess
 -      systemd-run -M ${DECK_UID}@ --user --collect --wait sh -c "./.steam/root/ubuntu12_32/steam steam://${command}/${encoded@Q}"
-+      @systemd@/bin/systemd-run -M ${DECK_UID}@ --user --collect --wait @handler@ "steam://${command}/${encoded@Q}"
++      @systemd@/bin/systemd-run -M ${DECK_UID}@ --user --collect --wait /run/current-system/sw/bin/steam "steam://${command}/${encoded@Q}"
```

Comment:
This is icky but it runs from udev :(

Reply:
Steam doesn't [or did not] need to be enabled systemwide for Jovian NixOS to work :thinking:.

Reply:
I guess let's just keep patching hwsupport, since it's a relatively cheap build.